mlflow/entities/experiment.py

```diff
@@ -6,7 +6,7 @@ class Experiment(_MLflowObject):
     """
     Experiment object.
     """
-    DEFAULT_EXPERIMENT_ID = 0
+    DEFAULT_EXPERIMENT_ID = "0"
```

Comment:
Should the default be here or live on the FileStore? Seems like we don't need all FileStores to share a default value once we aren't expecting them to all use integers.